ego/templates/deployment.yaml

```diff
@@ -147,6 +147,8 @@ spec:
               value: {{ .Values.appConfig.egoClientId }}
             - name: REACT_APP_KEYCLOAK_ENABLED
               value: {{ .Values.appConfig.uiKeycloakEnabled | quote }}
+            - name: REACT_APP_PASSPORT_ENABLED
+              value: {{ .Values.appConfig.uiPassportEnabled | quote }}
```

Comment:
you'll also have to add a default value for this variable here https://github.com/overture-stack/helm-charts/blob/d3e02e56b8ad404d40e719a7ccc631c4a93b5472/ego/values.yaml#L110

Reply:
thanks for the observation!! default value added as suggested.